modules/git/notes_gogit.go

```diff
@@ -10,19 +10,24 @@ import (
 	"context"
 	"io/ioutil"
 
+	"code.gitea.io/gitea/modules/log"
+
 	"github.com/go-git/go-git/v5/plumbing/object"
 )
 
 // GetNote retrieves the git-notes data for a given commit.
 func GetNote(ctx context.Context, repo *Repository, commitID string, note *Note) error {
+	log.Info("Searching for git note corresponding to the commit %q in the repository %q", commitID, repo.Path)
```

Comment:
I dont think we need this verbose logging - if you loke to leave it in we should use Trace as level

Reply:
Agreed. I'll change the severity of this message to Trace. @zeripath suggested that I add more logging to this function.

Reply:
I meant temporarily!

But making this one Trace could be helpful

Reply:
I've incorporated all of your comments